src/processing/mangekanter/mangekanter.md

```diff
@@ -0,0 +1,468 @@
+---
+title: Kanter, kanter, mange mangekanter
+level: 2

```

Comment:
Har du prøvd denne oppgaven på noen unger?

Helt fram til siste utfordring trur jeg det går bra. Likevel om det er mange nye konsept, trenger de kun kopiere koden. Og forståelsen vil nok i stor grad variere avhengig av om de faktisk tar seg tid til å lese koden. Men siste utfordring blir kanskje for meget? Altså fra "tegne pentagon".

Kunne "Tegne pentagon" egnet seg som en egen oppgave?